covenant-signer/docs/transition-from-phase1.md

```diff
@@ -0,0 +1,393 @@
+}
+```
+
+Parameters:
+- `<master-private-key>`: The base58-encoded master private key from your 
+Bitcoin wallet (first parameter)
+- `<derivation-path>`: The HD derivation path that specifies how to derive 
+the child key (second parameter)
+
+To verify, you can execute the following
+
+```shell
+bitdoind getaddressinfo <address> | jq .publickey
+```
+
+If the public key matches the derived_public_key output from the 
+`derive-child-key` command, the verification is successful.
+
+#### Step 4: Import the private key into a Cosmos Keyring
+
+Next, we are going to import the derived private key into the Cosmos keyring.
+At the moment, the `covenant-signer` has not implemented functionality
+for Cosmos key imports. To overcome this, we are going to utilize
+the Babylon node binary `babylond` which has support for them.
+You can install the binary by following the first section of the
+[Babylon Node Setup Guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/babylon-node/README.md).
+
+One you have the `babylond` binary installed,
+navigate to the directory you want to set up your keyring and run:
+
+```shell
+babylond keys import-hex cov fe1c56c494c730f13739c0655bf06e615409870200047fc65cdf781837cf7f06 \
+    --keyring-backend file \
+    --keyring-dir /path/to/your/keyring/directory
+```
+
+This command:
+- Uses `import-hex` to import the raw private key
+- Names the key `cov` in the keyring
+- Uses the secure `file` backend which encrypts the key on disk
+- Will prompt you for a passphrase to encrypt the key
+
+The passphrase you set here will be needed later on, keep this in mind.
+
+> ⚡ Note: While both `os` and `file` backends are supported, we recommend 
+using the `file` backend as it has been thoroughly tested across different 
+environments. The `file` backend stores the private key in encrypted form 
+on disk. When running `import-hex` with the `file` backend, you will be 
+prompted for a passphrase. This passphrase will be required to unlock the 
+signer later.
+
+To confirm that the import was successful, run:
+
+```shell
+babylond keys show cov
+```
+
+The output will display the details of the imported key:
+
+```shell
+    address: bbn1azasawj3ard0ffwj04zpxlw2pt9cp7kwjcdqmc
+    name: cov
+    pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ajp5tUbHnX98X/IGINkUtc9yUGMdEvbiZCftnT+Yxcyx"}'
+    type: local
+```
+
+Congratulations! You have successfully imported your key.
+
+## 4. Operation
+### 4.1. Configuration
+
+Next, we can return to the covenant signer directory 
+and create your own configuration file. Use the 
+following command to dump the configuration template:
+
+```shell
+covenant-signer dump-cfg --config <path-to-config-file>
+```
+
+This will create a configuration file, from the example configuration, 
+in the specified path.
+
+Replace the placeholder values with your own 
+configuration. This can be placed directly in the `covenant-signer` directory.
```

Comment:
What's the `covenant-signer` directory and what can be placed directly inside it?

If there's a need for a specific directory we need to instruct people to create it. If there's a need for a configuration file to be there, we should instruct them to place it there.

Reply:
But the covenant signer binary is in the global directory, no?